docs/resources/libraries.md

```diff
@@ -0,0 +1,208 @@
+- You can add new features and publish updated versions of the library. Each time a new version is published, the version number will automatically increment.
+
+:::
+
+## Importing a Library
+
+Open the FlutterFlow project where you'd like to import a library, then follow these steps:
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/DrzjKuhTWZXOxBB5yGJn?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+:::info
+
+- You can only select a library if you have been added as a collaborator in that library project. To use a library, you must have one of these roles in the library project: Owner, Manager, Editor, or Read-Only.
+- By default, the latest version of the library is imported, but you can choose to depend on an earlier version if needed.
+
+:::
+
+### Updating a Library
+
+You can easily upgrade to newer versions of the libraries as they become available. If a new update causes issues with your existing implementation, you also have the option to revert to a previous version.
+
+![update-library.avif](imgs/update-library.avif)
+
+### Manage Dependency Conflict while Import
+
+A **Dependency Conflict** occurs when two or more libraries added by a project depend on different versions of the same dependency. This creates a situation where the project cannot resolve which version to use, leading to a project error.
+
+![dependency-conflict.avif](imgs/dependency-conflict.avif)
+
+Let's say you are building an eCommerce app that uses multiple libraries for different purposes:
+
+- **User Auth Library** is used for handling user authentication.
+- **Payment Gateway Library** is used for managing the payment gateway.
+
+Both libraries depend on a common library called **Components Library** but with different versions:
+
+- **User Auth Library** depends on Components Library **v1.5.0**.
+- **Payment Gateway Library** depends on Components Library **v2.0.0**.
+
+In this scenario, eCommerce App project will detect the dependency conflict because it can't add both v1.5.0 and v2.0.0 of Components Library at the same time.
+
+#### Steps to Fix Dependency Conflict
+
+Follow these steps to ensure both libraries rely on the same version of Components Library:
+
+1. **Upgrade both libraries**: Start by upgrading libraries to their latest versions. Often, newer versions are built to rely on a common or more flexible version of Components Library, which can resolve conflicts.
```

Comment:
Little confusing here. Do you mean upgrading the dependencies used in these libraries? If yes, its not very clear here because of the heavy use of "libraries" and not mentioning which ones we are talking abiut here. Can we rephrase?

Reply:
Ya, It's updated now to make it more clear.